pkg/integrations/v2/apache_exporter/apache_exporter.go

```diff
@@ -0,0 +1,110 @@
+// Package apache_exporter embeds https://github.com/Lusitaniae/apache_exporter
+package apache_exporter //nolint:golint
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	ae "github.com/Lusitaniae/apache_exporter/collector"
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	integrations_v2 "github.com/grafana/agent/pkg/integrations/v2"
+	"github.com/grafana/agent/pkg/integrations/v2/common"
+	"github.com/grafana/agent/pkg/integrations/v2/metricsutils"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// DefaultConfig holds the default settings for the apache_exporter integration
+var DefaultConfig = Config{
+	ApacheAddr:         "http://localhost/server-status?auto",
+	ApacheHostOverride: "",
+	ApacheInsecure:     false,
+}
+
+// Config controls the apache_exporter integration.
+type Config struct {
+	ApacheAddr         string               `yaml:"scrape_uri,omitempty"`
+	ApacheHostOverride string               `yaml:"host_override,omitempty"`
+	ApacheInsecure     bool                 `yaml:"insecure,omitempty"`
+	Common             common.MetricsConfig `yaml:",inline"`
+}
+
+// ApplyDefaults applies the integration's default configuration.
+func (c *Config) ApplyDefaults(globals integrations_v2.Globals) error {
+	c.Common.ApplyDefaults(globals.SubsystemOpts.Metrics.Autoscrape)
+	return nil
+}
+
+// Identifier returns a string that identifies the integration.
+func (c *Config) Identifier(globals integrations_v2.Globals) (string, error) {
+	return *c.Common.InstanceKey, nil
+}
+
+// UnmarshalYAML implements yaml.Unmarshaler for Config
+func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	*c = DefaultConfig
+
+	type plain Config
+	return unmarshal((*plain)(c))
+}
+
+// Name returns the name of the integration this config is for.
+func (c *Config) Name() string {
+	return "apache"
```

Comment:
The v1 and v2 names should match (that's why CI is complaining)

Reply:
Ok, I decided to change name apache_http, as we tend to avoid _exporter suffixes, while just apache is too generic.